samples/js-prompts/src/index.ts

```diff
@@ -77,13 +78,19 @@ const threeGreetingsPrompt = ai.definePrompt(
 
 const threeGreetings = ai.defineFlow('threeGreetingsPrompt', async () => {
   const response = await threeGreetingsPrompt({ name: 'Fred' });
-  return response.output?.likeAPirate;
+  return response.output;
 });
 
 // Start a flow server, which exposes your flows as HTTP endpoints. This call
 // must come last, after all of your plug-in configuration and flow definitions.
 // You can optionally specify a subset of flows to serve, and configure some
 // HTTP server options, but by default, the flow server serves all defined flows.
-ai.startFlowServer({
+startFlowServer({
   flows: [threeGreetings, simpleTemplate, simpleDotprompt, simplePrompt],
 });
+
+// Call one of the flows just to validate everything is hooked up properly
```

Comment:
did you mean to leave this in?

Reply:
Yes, it makes the startup experience in IDX a little smoother if we output something from the main entry point